transport.validator.mapcss

```diff
@@ -1,10 +1,11 @@
 meta
 {
-	title: "Jungle Bus – validation ruleset";
-	version: "0.12";
+	title: "Jungle Bus - validation ruleset";
+	version: "0.13";
 	description: "Uncompromising validation of transit data";
 	author: "nlehuby";
 	link: "https://junglebus.io/";
+	license: "AGPL-3.0-or-later";
```

Comment:
the license is actually v3.0

Reply:
I was about to say, "the LICENSE file in the repo is the 3-or-later variation," but then I did some more reading and realized that I misunderstood the difference.

As it turns out, the proper way to do things is for the LICENSE file itself to merely describe version X, and *not* to specify whether the accompanying software is released under X-only or under X-or-later. That should be done in the header of each source code file, which don't currently exist in this repo. So it's currently ambiguous whether it's 3-only or 3-or-later.

To clear this up properly, you should first decide which of these you want it to be. If you want it to be 3-only, then make it that. If you want it to be 3-or-later, then you should reach out to the other five people who have contributed to this repo and ask their permission/clarification to make it 3-or-later. If you get a yes from *all* of them, then go ahead and do that; otherwise, you have to stick with 3-only.

Also, I mistakenly labelled it AGPL when it's actually GPL. :flushed: 

Also also, I haven't been able to find a standard/"correct" way to indicate a license in a `.mapcss` file, so I decided to just do it this way :shrug:. The [SPDX standard](https://spdx.dev/learn/handling-license-info/) would suggest instead having the very first line of the file be: `/* SPDX-License-Identifier: gplorwhatever */`, but that would make it more work to enable MapCSS software to read it. Although, now that I'm writing this, it would also make it *less* work (specifically, none) to enable general SPDX-related software to read it. Yeah, I've changed my mind: I prefer the first-line comment now.